addons/l10n_cl/views/res_partner.xml

```diff
@@ -15,11 +15,10 @@
             <field name="state_id" position="attributes">
                 <attribute name="placeholder">Región</attribute>
             </field>
-            <field name="country_id" position="after">
-                <field name="l10n_cl_sii_taxpayer_type"/>
+            <field name="vat" position="after">
+                <field name="l10n_cl_sii_taxpayer_type" placeholder="Tax Payer Type"/>
```

Comment:
Can we add attrs readonly when parent_id != False ?

Reply:
@Danisan

Reply:
> Can we add attrs readonly when parent_id != False ?

@jco-odoo you mean in the l10n_cl_sii_taxpayer_type? I am thinking that in general would be OK.

Reply:
When tested the readonly found that the value is not shown and only the label... why not invisible instead of readonly?

Reply:
made at 7f4e191e

Reply:
@jco-odoo

Reply:
@Danisan attrs="{'readonly': [('parent_id', '!=', False)]}"   Can you add it?  (because the vat number needs to be the sam for all the "children partners" of a partner)

Reply:
sorry, fixed at 1a33ac